lib/phase3-build-root.sh

```diff
@@ -136,8 +136,10 @@ eexec emerge $EMERGE_OPTS "sys-kernel/gentoo-sources"
 
 einfo "Installing genkernel..."
 
+echo "sys-kernel/genkernel -firmware" > /etc/portage/package.use/genkernel
```

Comment:
Looks like the change should be under `if eoff "$GENTOO_SYSTEMD"; then` condition.

Reply:
Good point, I've updated the PR with this change.